src/nvim/api/vim.c

```diff
@@ -299,7 +299,7 @@ ArrayOf(String) nvim_list_runtime_paths(void)
     FUNC_API_SINCE(1)
 {
   Array rv = ARRAY_DICT_INIT;
-  uint8_t *rtp = p_rtp;
+  char_u *rtp = p_rtp;
```

Comment:
I was already commenting this. `uint8_t` is 8-bit unsigned *number*. `char_u` is an unsigned *character*. It is deprecated, but not in a sense “you should replace char_u with uint8_t”, it is to be replaced by “char”.